lib/item.rb

```diff
@@ -1,3 +1,8 @@
 class Item
+    attr_reader :name, :price
 
+    def initialize(attrs)
+        @name = attrs[:name]
+        @price = attrs[:price]
```

Comment:
The interaction pattern asks to clean the price string (remove dollar sign) and convert to a float.